models/migrations/migrations.go

```diff
@@ -286,6 +286,8 @@ var migrations = []Migration{
 	NewMigration("Recreate user table to fix default values", recreateUserTableToFixDefaultValues),
 	// v169 -> v170
 	NewMigration("Update DeleteBranch comments to set the old_ref to the commit_sha", commentTypeDeleteBranchUseOldRef),
+	// v170 -> v171
+	NewMigration("Add sessions table for macaron/session", addSessionTable),
```

Comment:
Now the name changed to `go-chi/sessions`

Reply:
```suggestion
	NewMigration("Add sessions table for go-chi/session", addSessionTable),
```